src/scancode/__init__.py

```diff
@@ -41,3 +43,11 @@
 
 if not exists(scans_cache_dir):
     fileutils.create_dir(scans_cache_dir)
+
+
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    __version__ = get_distribution('scancode-toolkit').version
+except DistributionNotFound:
+    # package is not installed ??
+    __version__ = '2.0.0rc3'
```

Comment:
This is what should happen, but I am playing it safe. The versions are updated at once in all places with `bumpversion` in anycase